docs/running-local.md

```diff
@@ -0,0 +1,236 @@
+Running On a Local (Big) Machine
+================================
+
+Prerequisites
+-------------
+
+You need a beefy Linux `x86_64` machine, with `docker` and `docker-compose`
+available (and ready to use by the current UNIX user).
```

Comment:
By default, on many systems `docker` cannot be used by a normal Unix-user, that's why I added that precision and also in the Ubuntu example I added the `adduser <user> docker` to give the right access rights.